cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,48 @@
+/// <reference types='cypress' />
+
 describe('', () => {
-  before(() => {
+  let user;
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.login(user.email, user.username, user.password);
+      cy.visit('/');
+    });
   });
 
-  it('', () => {
-
+  it('should have the ability to create an article', () => {
+    let article;
+    let usernameToLowerCase = user.username.toLowerCase();
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+      cy.createArticle(article.title, article.about, article.body);
+      cy.get('.link.nav-link').eq(1).click();
+      cy.contains('.article-preview', article.title).should('exist');
+      cy.contains('.article-preview', article.title)
+        .find('.author')
+        .should('have.text', usernameToLowerCase);
+    });
   });
 
-  it('', () => {
+  it('should have the ability to delete an article', () => {
+    const textConfirmAlert = 'Do you really want to delete it?';
+    const messageForNoPosts = 'No articles are here... yet.';
+    let article;
+
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+      cy.createArticle(article.title, article.about, article.body);
```

Comment:
```suggestion
      cy.createArticle(article.title, article.about, article.body);
```
try to extract article slug from that request and visit directly the article page